tests.toml

```diff
@@ -0,0 +1,21 @@
+test_format = 1.0
+
+[default]
+
+    # ------------
+    # Tests to run
+    # ------------
+
+    exclude ="install.nourl"
+
+    # -------------------------------
+    # Default args to use for install
+    # -------------------------------
+
+    args.default_home_server="matrix.org"
+
+    # -------------------------------
+    # Commits to test upgrade from
+    # -------------------------------
+
+    test_upgrade_from.d653976.name = "Upgrade from 23.01"
```

Comment:
I don't know what commit you meant but apparently this one doesnt exists :sweat_smile: